dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -265,11 +351,21 @@ public fun DataFrame.Companion.readExcel(
     rowsCount: Int? = null,
     nameRepairStrategy: NameRepairStrategy = NameRepairStrategy.CHECK_UNIQUE,
     firstRowIsHeader: Boolean = true,
+    parseEmptyAsNull: Boolean = true,
 ): AnyFrame {
     val sheet: Sheet = sheetName
         ?.let { wb.getSheet(it) ?: error("Sheet with name $sheetName not found") }
         ?: wb.getSheetAt(0)
-    return readExcel(sheet, columns, formattingOptions, skipRows, rowsCount, nameRepairStrategy, firstRowIsHeader)
+    return readExcel(
+        sheet,
+        columns,
+        formattingOptions,
+        skipRows,
+        rowsCount,
+        nameRepairStrategy,
+        firstRowIsHeader,
+        parseEmptyAsNull,
```

Comment:
My 5 cents: how about enum parameter "emptyStringValue" with NULL, KEEP_AS_STRING options?

Reply:
or, if we're looking to extend this with other null strings https://github.com/Kotlin/dataframe/issues/1248, it could be some sort of sealed interface (String) -> String? with companion object { val NULL = ...; val KEEP_AS_STRING ... }

Reply:
I'd like something like this too, but it would deviate even more for how we handle parsing `String -> null` with `parse()`, `ParserOptions`, and `readCsv()`. They now take a simple set of null-representing strings.
Maybe we could change it all to the same system at some point, but until then I'd keep this one as simple as possible.

Reply:
what about enum then?